vault-1.14.advisories.yaml

```diff
@@ -139,6 +139,11 @@ advisories:
             componentType: go-module
             componentLocation: /usr/bin/vault
             scanner: grype
+      - timestamp: 2024-05-04T17:59:28Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: 'A govulncheck analysis found that none of the known affected symbols were present in the APK, after examining the following paths within the package: /usr/bin/vault.'
```

Comment:
This remains me why we don't enable govulncheck by default when scanning our packages.

Reply:
Perhaps we need to enable by default.

Reply:
Yeah I think somehow if we can automate triaging that'd be an awesome next step!